### PR TITLE
docs(readme): add sinnerschrader to readme

### PR DESCRIPTION
**Pull request by tilmx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,4 +140,4 @@ Alva follows the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md).
 
 ---
 
-Copyright 2017. Released under the MIT license.
+Copyright 2017. Proudly powered by [SinnerSchrader](https://github.com/sinnerschrader). Released under the MIT license.
```
Comment (LKuechler) [inline README.md]: maybe move it on its own line above the copyright

Comment (tilmx) [reply README.md]: I actually like it in the footer. The section above should be more about contributing.

Comment (LKuechler) [reply README.md]: What i meant was that is should not be in line with the Copyright :)

**Comment by tilmx** [comment]
Just moved it up.

**Pull request merged**